Source/RealmQueueManager.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  RealmQueueManager.swift
+//  RealmResultsController
+//
+//  Created by Pol Quintana on 09/12/15.
+//  Copyright © 2015 Redbooth. All rights reserved.
+//
+
+import Foundation
+
+struct RealmQueueManager {
+    let operationQueue: NSOperationQueue = {
+        let queue = NSOperationQueue()
+        queue.maxConcurrentOperationCount = 1
+        queue.name = "com.RRC.\(arc4random_uniform(1000))"
+        return queue
+    }()
+    
+    func addOperation(withBlock block: ()->()) {
+        guard !Threading.isTesting else {
```

Comment:
Can not be this outside of `RealmQueueManager`? Not sure that this struct and `Threading` have to know about the current environment